examples/attach/test.bash

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -o pipefail -o errexit -o nounset
 
+export HOME="$TEST_TMPDIR"
```

Comment:
Should there be an added assertion?

Reply:
I don't think we need one for this. it is there because cosign wants to read credentials which unrestricted at `bazel run`